### PR TITLE
Add test cases for EventListenerToEventSubscriberRector

### PR DESCRIPTION
**Pull request by flohw**
This should solve #4899 

Thank you :slightly_smiling_face:

### CODE REVIEW

File: rules/symfony-code-quality/tests/Rector/Class_/EventListenerToEventSubscriberRector/Fixture/multiple_listeners_called_once.php.inc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Rector\SymfonyCodeQuality\Tests\Rector\Class_\EventListenerToEventSubscriberRector\Fixture;
+
+class MultipleMethodsCalledOnce
```
Comment (samsonasik) [inline rules/symfony-code-quality/tests/Rector/Class_/EventListenerToEventSubscriberRector/Fixture/multiple_listeners_called_once.php.inc:5]: It need a name based on file name `multiple_listeners_called_once.php.inc` -> `class MultipleListenersCalledOnce`.

There is `validate-fixture-classname` command that can automatically fix that, but currently not runs for `rules` directory due many false positive.

```suggestion
class MultipleListenersCalledOnce
```

File: rules/symfony-code-quality/tests/Rector/Class_/EventListenerToEventSubscriberRector/Fixture/multiple_listeners_called_once.php.inc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\SymfonyCodeQuality\Tests\Rector\Class_\EventListenerToEventSubscriberRector\Fixture;
+
+class MultipleMethodsCalledOnceEventSubscriber implements \Symfony\Component\EventDispatcher\EventSubscriberInterface
```
Comment (samsonasik) [inline rules/symfony-code-quality/tests/Rector/Class_/EventListenerToEventSubscriberRector/Fixture/multiple_listeners_called_once.php.inc:22]:
```suggestion
class MultipleListenersCalledOnceEventSubscriber implements \Symfony\Component\EventDispatcher\EventSubscriberInterface
```

**Comment by flohw** [comment]
oops, I didn't double check. I applied the changes you suggested. @samsonasik 

Thank you

File: rules/symfony-code-quality/tests/Rector/Class_/EventListenerToEventSubscriberRector/config/listener_services.xml [hunk 3]
```diff
@@ -15,6 +15,16 @@
             <tag name="kernel.event_listener" event="multi_event" method="meToo"/>
         </service>
 
+        <service id="fourth_listener" class="Rector\SymfonyCodeQuality\Tests\Rector\Class_\EventListenerToEventSubscriberRector\Fixture\MultipleMethodsCalledOnce">
```
Comment (samsonasik) [inline rules/symfony-code-quality/tests/Rector/Class_/EventListenerToEventSubscriberRector/config/listener_services.xml:18]:
```suggestion
        <service id="fourth_listener" class="Rector\SymfonyCodeQuality\Tests\Rector\Class_\EventListenerToEventSubscriberRector\Fixture\MultipleListenersCalledOnce">
```

**Comment by flohw** [comment]
Chaotic first PR ^^ 

Do you have everything you need @samsonasik or other changes are needed?

Thank you

**Comment by samsonasik** [comment]
it seems ok now

**Comment by TomasVotruba** [comment]
@samsonasik Thank you! Could you handle the fix too?

**Comment by samsonasik** [comment]
I will try.

**Comment by samsonasik** [comment]
@flohw I chery-picked your commits at PR https://github.com/rectorphp/rector/pull/4908

**Comment by flohw** [comment]
Fixed in #4904 

Thank you :slightly_smiling_face:

**Pull request closed without merging**